llvm/lib/SYCLLowerIR/LowerESIMD.cpp

```diff
@@ -372,7 +372,14 @@ class ESIMDIntrinDescTable {
         {"cos", {"cos", {a(0)}}},
         {"pow", {"pow", {a(0), a(1)}}},
         {"div_ieee", {"ieee.div", {a(0), a(1)}}},
-        {"dp4a", {"dp4a", {a(0), a(1), a(2)}}},
+        {"uudp4a", {"uudp4a", {a(0), a(1), a(2)}}},
```

Comment:
please add a unit test. See https://github.com/intel/llvm/blob/sycl/llvm/test/SYCLLowerIR/esimd_lower_intrins.ll

Reply:
I have pushed end-to-end tests to llvm_test_suite PR#75, which should provide better validation coverage.

Also added lowering unit test in new commit here.

Reply:
> I have pushed end-to-end tests to llvm_test_suite PR#75, which should provide better validation coverage.

It is a requirement that LLVM pass updates come with unit tests. This way it is runnable with just opt.

> Also added lowering unit test in new commit here.

Thanks!

Reply:
Got it. I only see you as the owner. Will this be auto merged, or do I need to contact someone else?